### PR TITLE
Implement Noizd transformer

### PR DESCRIPTION
**Pull request by Kunkka0822**

### CODE REVIEW

File: test/strategies/noizd-get-tokenuri/transformer_test.mjs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import { onLine } from "../../../src/strategies/soundxyz-get-tokenuri/transformer.mjs";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const snapshot = JSON.parse(
+  fs.readFileSync(resolve(__dirname, "./extractor_snapshot.json"))
+);
+
+test("soundxyz-get-tokenuri transformer", (t) => {
```
Comment (TimDaub) [inline test/strategies/noizd-get-tokenuri/transformer_test.mjs:16]:
```suggestion
test("noiz-get-tokenuri transformer", (t) => {
```

File: test/strategies/noizd-call-tokenuri/transformer_test.mjs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import { onLine } from "../../../src/strategies/soundxyz-call-tokenuri/transformer.mjs";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const snapshot = JSON.parse(
+  fs.readFileSync(resolve(__dirname, "./extractor_snapshot.json"))
+);
+
+test("soundxyz-call-tokenuri transformer", (t) => {
```
Comment (TimDaub) [inline test/strategies/noizd-call-tokenuri/transformer_test.mjs:16]: rename

**Comment by neatonk** [comment]
It looks like the checks failed because the github action expects the branch to exist in this repo instead of @Kunkka0822's fork.

https://github.com/neume-network/strategies/actions/runs/3107105334/jobs/5035690711#step:2:455

**Comment by il3ven** [comment]
The PR looks good to me. I will fix the github actions issue, run the tests and then merge the PR.

**Comment by Kunkka0822** [comment]
> It looks like the checks failed because the github action expects the branch to exist in this repo instead of @Kunkka0822's fork.
> 
> https://github.com/neume-network/strategies/actions/runs/3107105334/jobs/5035690711#step:2:455

How can I fix this problem?

**Comment by il3ven** [comment]
> It looks like the checks failed because the github action expects the branch to exist in this repo instead of @Kunkka0822's fork.
> 
> https://github.com/neume-network/strategies/actions/runs/3107105334/jobs/5035690711#step:2:455

I fixed it in https://github.com/neume-network/strategies/commit/450f646b4e7661d9e82690704ce5a1d9d1830b48.

@TimDaub The _unit tests_ GitHub actions are not able to run because PRs don't have access to secrets. I couldn't find any solution for this. There are only hacks like [ok-to-test](https://github.com/imjohnbo/ok-to-test). We can either manually test external contributions or remove secrets and use a public RPC node like https://rpc.ankr.com/eth.

That said, I have manually tested this PR and merging it now. Thanks @Kunkka0822.

**Pull request merged**